### PR TITLE
Added Navigation Drawer

### PR DESCRIPTION
**Pull request by SachinRammoorthy**

### CODE REVIEW

File: app/src/main/java/com/community/jboss/leadmanagement/MainActivity.java [hunk 1]
```diff
@@ -1,13 +1,107 @@
...
+    private Toolbar toolbar;
+    private NavigationView navigationView;
+    private DrawerLayout drawerLayout;
+ 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
+     
+     toolbar = (Toolbar) findViewById(R.id.toolbar);
```
Comment (garg-anuj) [inline app/src/main/java/com/community/jboss/leadmanagement/MainActivity.java]: Please use butterknife for these bindings http://jakewharton.github.io/butterknife/

File: app/src/main/java/com/community/jboss/leadmanagement/MainActivity.java [hunk 2]
```diff
@@ -1,13 +1,110 @@
...
+    private Toolbar toolbar;
+    private NavigationView navigationView;
+    private DrawerLayout drawerLayout;
+ 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
+     
+     @BindView(R.id.toolbar) Toolbar toolbar;
```
Comment (garg-anuj) [inline app/src/main/java/com/community/jboss/leadmanagement/MainActivity.java]: move these to instance variables

**Comment by garg-anuj** [comment]
@SachinRammoorthy You are not running your code locally before commit and push?

**Comment by garg-anuj** [comment]
`Verified` in front is saying you have made all changed manually adding in github online.

**Comment by SachinRammoorthy** [comment]
@garg-anuj of course i run it locally!

File: app/src/main/java/com/community/jboss/leadmanagement/MainActivity.java [hunk 3]
```diff
@@ -1,13 +1,112 @@
...
+ 
 public class MainActivity extends AppCompatActivity {
-
+ 
+    //Defining Variables
+    /*private Toolbar toolbar;
+    private NavigationView navigationView;
+    private DrawerLayout drawerLayout; */
+ 
+      @BindView(R.id.toolbar) Toolbar toolbar;
```
Comment (garg-anuj) [inline app/src/main/java/com/community/jboss/leadmanagement/MainActivity.java]: `BindView` is not imported

File: app/src/main/java/com/community/jboss/leadmanagement/MainActivity.java [hunk 4]
```diff
@@ -1,13 +1,112 @@
...
+      @BindView(R.id.nav_slide) NavigationView navigationView;
+      @BindView(R.id.drawer_layout) DrawerLayout drawerLayout;
+ 
+ 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
+     
+      //Toolbar toolbar = (Toolbar) findViewById(R.id.toolbar);
```
Comment (garg-anuj) [inline app/src/main/java/com/community/jboss/leadmanagement/MainActivity.java]: Add
```
ButterKnife.bind(this);
```

File: app/src/main/java/com/community/jboss/leadmanagement/MainActivity.java (hunk 3 above)
Comment (SachinRammoorthy) [reply app/src/main/java/com/community/jboss/leadmanagement/MainActivity.java]: @garg-anuj done

File: app/src/main/java/com/community/jboss/leadmanagement/MainActivity.java (hunk 4 above)
Comment (SachinRammoorthy) [reply app/src/main/java/com/community/jboss/leadmanagement/MainActivity.java]: Added and updated

**Comment by garg-anuj** [comment]
Thanks for your awesome work @SachinRammoorthy  🎉 🎉 🎉 🎉 🎉

**Pull request closed without merging**